src/main/java/de/retest/recheck/Properties.java

```diff
@@ -2,9 +2,12 @@
 
 public class Properties {
 
-	public static final String FILE_OUTPUT_FORMAT_PROPERTY = "de.retest.output.Format";
+	public static final String STATE_OUTPUT_FORMAT_PROPERTY = "de.retest.output.Format";
+	public static final String REPORT_OUTPUT_FORMAT_PROPERTY = "de.retest.recheck.output.reportFormat";
 	public static final String CONFIG_FILE_PROPERTY = "de.retest.configFile";
 
+	public static final String REHUB_ENABLED = "de.retest.recheck.rehub.enabled";
```

Comment:
Do we now have two properties that serve the same purpose?

1. `de.retest.recheck.output.reportFormat=cloud` enables rehub?
2. `de.retest.recheck.rehub.enabled=cloud` enables rehub?

What if the report format differs from the rehub flag?

Reply:
I think `REHUB_ENABLED` is more expressive than setting just a (technical) detail like using `FileOutputFormat.CLOUD`.

Reply:
Totally agree, but since we expose both properties to the user, this could cause conflicting situations and might confuse. This is also error prone if one class uses property A and another one property B.